buildenv

```diff
@@ -138,7 +138,9 @@ zopen_install()
 
 zopen_append_to_setup() {
 cat <<ZZ
-  \$PWD/bin/zopen init --refresh
+  if [ -n "\$ZOPEN_ROOTFS" ] && [ -d "\$ZOPEN_ROOTFS" ]; then
+    \$PWD/bin/zopen init --refresh
```

Comment:
do you want to verify that \$PWD/bin/zopen is executable first in case we move stuff around in the future and break this layout?

Reply:
There are probably other places we have this assumption so ok if we don't do this one - but seems a bit dangerous without a check

Reply:
Yeah, curl, git, and a few others :) Although we have checks in place for the return code there: https://github.com/ZOSOpenTools/gitport/blob/main/buildenv#L109